src/time/span.cr

```diff
@@ -59,7 +59,7 @@ struct Time::Span
 
   def self.new(days : Int, hours : Int, minutes : Int, seconds : Int, nanoseconds : Int = 0)
```

Comment:
I currently have kept the old initialization methods.
Is desirable to keep them, and if so, do I have to do something about adding the deprecated status?

Reply:
This overload should probably be removed because it shadows the new one. Or maybe just prefix every argument with `_` and add `@[Deprecated]` annotation.

Reply:
This overload should probably be removed because it shadows the new one. Or maybe just prefix every argument with `_` and add `@[Deprecated]` annotation.

Reply:
I added `@[Deprecated]` annotations with a message to use `#new` with named args instead